testbed/test_subtask_span_propagation/README.md

```diff
@@ -4,7 +4,7 @@ This example shows an active `Span` being simply propagated to the subtasks -eit
 
 Implementation details:
 - For `threading`, `gevent` and `asyncio` the `Span` is manually passed down the call chain, being manually reactivated it in each corotuine/task.
-- For `tornado`, the active `Span` is not passed nor activated down the chain as the custom `StackContext` automatically propagates it.
+- For `tornado` and `contextvars`, the active `Span` is not passed nor activated down the chain due to implicitly context propagating.
```

Comment:
```suggestion
- For `tornado` and `contextvars`, the active `Span` is not passed down the chain nor activated because the context is implicitly propagated.
```